src/parser.rs

```diff
@@ -405,18 +405,27 @@ impl<'tokens, 'src> Parser<'tokens, 'src> {
     let expression = if self.accepted_keyword(Keyword::If)? {
       self.parse_conditional()?
     } else {
-      let value = self.parse_value()?;
-
+      let value = self.parse_value();
       if self.accepted(Slash)? {
+        // Enables Slash operator to build absolute paths, like `/ "Users"`, by
+        // setting a default empty string on the lhs if the resulting expression
+        // is not Ok.
+        let value = value.unwrap_or(Expression::StringLiteral {
+          string_literal: StringLiteral {
+            kind: StringKind::new(string_kind::StringDelimiter::QuoteDouble, false),
```

Comment:
I also noticed that something like this is possible:

```rs
kind: StringKind::from_token_start("\"").unwrap()
```

I didn't do it, however, because it looked a bit hacky to me, although it would mean that nothing from the `string_kind` module would need to be made public.